README.md

```diff
@@ -40,3 +40,12 @@ lastly, you would need to boot ipxe, a simple way to test on a vm, while ipxe is
 dhcp
 chain <your base url>/ipxe
 ```
+
+## Use as script
+
+If you cannot run the container, you can run the script directly:
+
+```
+$ BASE_URL=http://webhost.local/ ./iso_to_ipxe path/to/image.iso
+```
```

Comment:
note the space in the url?

Reply:
There's a space between the URL and the command to run the script. It could alternatively be two lines:

```
$ export BASE_URL=http://webhost.local/
$ ./iso_to_ipxe path/to/image.iso
```